assets/sites/default/settings.acquia.php

```diff
@@ -5,6 +5,29 @@
  * Acquia Settings.
  */
 
+/**
+ * Override domain detection in Acquia Purge.
+ */
+if (isset($_ENV['AH_SITE_ENVIRONMENT'])) {
+  switch ($_ENV['AH_SITE_ENVIRONMENT']) {
+    case 'prod':
+      $domain = $conf['acquia']['prod']['base_url'];
+      $domain = substr($domain, 7);
```

Comment:
@janette It is possible that the base_urls will never be given in a different format, but I rather we do not make that assumption. I believe `substr($domain, 7);` is removing http:// from the base_url, but that feels very fragile. Maybe we should split the screen on "://" and take the second chunk, that will make no assumption about the protocol.